sympy/assumptions/refine.py

```diff
@@ -348,5 +371,6 @@ def refine_sign(expr, assumptions):
     'StrictLessThan': refine_Relational,
     're': refine_re,
     'im': refine_im,
-    'sign': refine_sign
-}  # type: Dict[str, Callable[[Expr, Boolean], Expr]]
+    'sign': refine_sign,
+    'matrixelement': refine_matrixelement
```

Comment:
Why would you name the handler with lowercase letters?
I think that it should align with the class name

Reply:
I have replaced `matrixelement` with `MatrixElement` is it okay?

Reply:
For these two lines of code, it looks fine, but do check the other parts of code for lowercase